src/hotspot/share/opto/opaquenode.hpp

```diff
@@ -70,38 +70,23 @@ class OpaqueLoopStrideNode : public Opaque1Node {
   virtual int Opcode() const;
 };
 
-//------------------------------Opaque2Node------------------------------------
-// A node to prevent unwanted optimizations.  Allows constant folding.  Stops
-// value-numbering, most Ideal calls or Identity functions.  This Node is
-// specifically designed to prevent the pre-increment value of a loop trip
-// counter from being live out of the bottom of the loop (hence causing the
-// pre- and post-increment values both being live and thus requiring an extra
-// temp register and an extra move).  If we "accidentally" optimize through
-// this kind of a Node, we'll get slightly pessimal, but correct, code.  Thus
-// it's OK to be slightly sloppy on optimizations here.
```

Comment:
Not sure if we should also add this comment again to the `Opaque3` nodes as it suggests that it is okay to be sloppy with `Opaque3` as well (but then on the other hand, what exactly is the definition of being sloppy in the context of `Opaque3` nodes? So, it might be more confusing than actually helping).

Reply:
I would leave it out, unless if we are sure that it really applies to `Opaque3` Nodes.